src/mbgl/util/dtoa.cpp

```diff
@@ -1,10 +1,15 @@
 #include "dtoa.hpp"
 
+// Clang on Windows 64-bits can't parse rapidjson's dtoa
```

Comment:
I will add more details on the comments. MSVC Clang/C2 frontend was crashing here. We can probably revisit this `#ifdef` after they release a new update.